teamB/jeongyunah/controllers/postController.js

```diff
@@ -0,0 +1,93 @@
+const postService = require('../services/postService')
+
+const postUp = async (req, res) => {
```

Comment:
게시물 생성과 관련된 함수일까요?? 그렇다면 다음과 같이 액션과 대상의 조합으로 즉, 동사형으로 만들어 주면 좋을 것 같습니다.
```suggestion
const createPost = async (req, res) => {
```